lib/schedule/gtfs/stop.ex

```diff
@@ -44,6 +48,96 @@ defmodule Schedule.Gtfs.Stop do
     }
   end
 
+  @spec reject_connections_for_route(t(), Route.id()) :: t()
+  @doc """
+  Remove any stop connections with the given route_id
+  """
+  def reject_connections_for_route(stop, route_id) do
+    %{stop | connections: Enum.reject(stop.connections, &(&1.id == route_id))}
+  end
+
+  @doc """
+  Add connections to a map of stops by id based on the given route, pattern, and stop time data.
+  The list of connections for a stop will include connections for all sibling and parent stops.
+  """
+  @spec stops_with_connections(by_id(), [Route.t()], [RoutePattern.t()], StopTime.by_trip_id()) ::
+          by_id()
```

Comment:
praise: thanks for the doc comments!

comment: I liked the `\@spec` being at the top of the `\@doc` in the code like on [reject_connections_for_route](https://github.com/mbta/skate/blob/31f2a3041c8fc810b1e95cebd023476a79c7af4c/lib/schedule/gtfs/stop.ex#L51), but I don't have strong feelings about this one way or the other besides trying to decide what style we should use?